Library/Formula/geocouch.rb

```diff
@@ -78,7 +78,7 @@ def install
     (couchdb_share/"www/script/test/geocouch").mkpath
     (couchdb_share/"www/script/test/geocouch").install test_files
     Dir[(couchdb_share/"www/script/test/geocouch/*.js")].each  \
-      { |geotest| system "cd #{couchdb_share/"www/script/test"};  ln -s geocouch/#{File.basename(geotest)} ." }
+      { |geotest| Dir.chdir("#{couchdb_share/"www/script/test"}"); ln("ln -s geocouch/#{File.basename(geotest)} .") }
```

Comment:
This changes the original behavior which was to make a symlink without changing the current process’ directory.